meetings/bordeaux2018/index.md

```diff
@@ -44,13 +44,41 @@ A preliminary schedule is outlined below. It will be updated once speakers are c
 
 ### Wednesday 27 June
 
-<tt>13:00</tt> Workshop
+<tt>13:30 - 14:00</tt> Introduction **Juan-Pablo Vielma (MIT)**
 
-<tt>17:00</tt> Finish
+<tt>14:00 - 14:25</tt> Systematically building mixed-integer programming formulations using JuMP and Julia **Joey Huchette (MIT)**
+
+<tt>14:25 - 14:50</tt> Modeling decomposable Mixed Integer Programs **Guillaume Marques (Université de Bordeaux)**
+
+<tt>14:50 - 15:15</tt> Coffee Break
+
+<tt>15:15 - 15:40</tt> [Artelys Knitro](https://www.artelys.com/en/optimization-tools/knitro) 11.0, a new conic solver and other novelties **Jean-hubert Hours (Artelys)**
+
+<tt>15:40 - 16:05</tt> Power and exponential Cones with [Mosek](https://www.mosek.com/) **Ulf Worsøe (MOSEK)**
+
+<tt>16:05 - 16:30</tt> [ProxSDP.jl](https://github.com/mariohsouto/ProxSDP.jl): A semidefinite programming solver written in Julia **Joaquim Dias Garcia & Mario Souto (PUC-Rio)**
+
+<tt>16:30</tt> Finish
 
 ### Thursday 28 June
 
-<tt>Day</tt> Workshop
+<tt>10:00 - 10:25</tt> Stochastic Programming for Hydropower Operations: Modeling and Algorithms **Martin Biel (KTH - The Royal Institute of Technology)**
+
+<tt>10:25 - 10:50</tt> Optimal energy management and stochastic decomposition **François Pacaud (CERMICS, ENPC)**
+
+<tt>10:50 - 11:20</tt> Coffee Break
+
+<tt>11:20 - 11:45</tt> A Julia JuMP-based module for polynomial optimization with complex variables applied to Optimal Power Flow  **Julie Sliwak (RTE)**
+
+<tt>11:45 - 12:05</tt> [POD](https://github.com/lanl-ansi/POD.jl), A Global Solver for Nonconvex MINLPs **Harsha Nagarajan (LANL)**
+
+<tt>12:05 - 12:40</tt> [The Gravity modeling language](https://www.allinsights.io/gravity) **Hassan Hijazi (LANL)**
+
+<tt>12:40 - 14:00</tt> Lunch
+
+<tt>14:00 - 14:30</tt> MathOptInterface and JuMP 0.19 **Miles Lubin (Google)**
```

Comment:
We can could schedule this for 45 min or an hour. Doesn't matter too much given there's nothing after.